optional/bed_mesh.cfg

```diff
@@ -42,11 +42,18 @@ gcode:
       {% set mesh_max_y = safe_max_y %}
     {% endif %}
 
-    {% set max_x_probes = (params.PROBE_COUNT |
+    # Klipper allows probe_count to be a single integer, this handles that case
+    {% if ',' in bed_mesh.probe_count %}
+      {% set max_x_probes = (params.PROBE_COUNT |
                            default(bed_mesh.probe_count)).split(",")[0]|int %}
-    {% set max_y_probes = (params.PROBE_COUNT |
+      {% set max_y_probes = (params.PROBE_COUNT |
                            default(bed_mesh.probe_count)).split(",")[1]|int %}
```

Comment:
Good catch, but I think it would be better to leave the original code unmodified and just change the above line to something like this:

```
                           (default(bed_mesh.probe_count)).split(",")[1]|int|
                            default(max_x_probes)) %}
```

Seems to me this would accomplish the same thing, but is a simpler change.